sys/hashes/sha256.c

```diff
@@ -259,3 +259,62 @@ unsigned char *sha256(const unsigned char *d, size_t n, unsigned char *md)
 
     return md;
 }
+
+unsigned char * hmac_sha256(const unsigned char *key,
+                            size_t key_length,
+                            const unsigned *message,
+                            size_t message_length,
+                            unsigned char *result)
+{
+    unsigned char k[64];
```

Comment:
Can you replace 64 by a macro constant? (Could also go into a separate PR if you like.)